packages/client/hmi-client/src/components/home/tera-project-menu.vue

```diff
@@ -58,11 +60,31 @@ const copyMenuItem = {
 	}
 };
 
+const downloadMenuItem = {
+	label: 'Download this project',
+	icon: 'pi pi-clone',
+	command: async () => {
+		if (props.project) {
+			isCopying.value = true;
+			const blob = await exportProjectAsFile(props.project.id);
+			if (blob) {
+				const a = document.createElement('a');
+				a.href = URL.createObjectURL(blob);
+				a.download = `${props.project.name}.${AcceptedExtensions.PROJECTCONFIG}`;
+				a.click();
+				a.remove();
+			}
+			isCopying.value = false;
+		}
+	}
+};
+
 const separatorMenuItem = { separator: true };
 const projectMenuItems = computed(() => {
 	const items = [] as any[];
```

Comment:
If this is easy to add in try:

```suggestion
	const items: MenuItem[] = [];
```

and

```
import { MenuItem } from 'primevue/menuitem';
```